src/main/java/client/Client.java

```diff
@@ -0,0 +1,29 @@
+package main.java.client;
+
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+
+public class Client {
+
+    private static final int PORT = 8080;
+    private static final String HOST = "127.0.0.1";
+
+
+    public static void main(String[] args) {
+        try {
+            Socket socket = new Socket(HOST, PORT);
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Use try-with-resources or close this "Socket" in a "finally" clause. ([squid:S2095](https://sonarcloud.io/organizations/jangchulwoon-github/rules#rule_key=squid%3AS2095))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=groupId%3AHTTPServer&pullRequest=1&issues=AWQTU_q-bXXTsx7O43dU&open=AWQTU_q-bXXTsx7O43dU)

[ISSUE_KEY]: AWQTU_q-bXXTsx7O43dU